Assets/Code/SceneManager.cs

```diff
@@ -26,13 +28,10 @@ void Main ()
     void Start()
     { 
         Debug.Log("Starting");
-        // PositionGrid testGrid = new PositionGrid();
         Instantiate(testCharacter, charPosition, Quaternion.identity);
-    }
-
-    // Update is called once per frame
-    void Update()
-    {
-        
+        //testCharacter.UpdateSprite();
+        Instantiate(testEnemy, enemyPosition, Quaternion.identity);
+        //testEnemy.UpdateSprite();
+        //testEnemy.Position = new GridCoordinates(5, 1);
```

Comment:
It looks like these commented out lines were implementations that you tried and then opted not to use. Unless you have a reason for leaving them here we should just delete them to tidy up the code.

Reply:
Those comments should have been removed in https://github.com/darin-ellis7/GridGame/issues/8